packages/backend/src/models/ResourceViewItemModel.ts

```diff
@@ -32,6 +36,7 @@ const getCharts = async (
                 'saved_queries.last_version_updated_by_user_uuid',
             order: 'pinned_chart.order',
             chart_kind: `saved_queries.last_version_chart_kind`,
```

Comment:
instead of returning chart_type, should we return a valid chart_kind ? is it because it is not stored correctly in db ? this way we can avoid making a unnecessary join, which will slow down the call.

Reply:
`chart_type` is actually not needed here, only `chart_kind`
The problem was that we were returning `chartType: row.chart_kind` instead of `chartKind: row.chart_kind`, so I can remove the join 👌